es6/003_arrow_function/arrowFunction.js

```diff
@@ -0,0 +1,11 @@
+// Mendeklarasikan dan menginisialisasi array ke dalam variabel
+const islands = [
+    'Sumatera',
+    'Jawa',
+    'Kalimantan',
+    'Sulawesi',
+    'Papua'
```

Comment:
Mesti make doublequute, jangan single quute. Easy fix jalanin prettiernya

```bash
npm run prettier
```

```suggestion
    "Sumatera",
    "Jawa",
    "Kalimantan",
    "Sulawesi",
    "Papua"
```

Reply:
siap gass